spec/models/cost_query/filter_spec.rb

```diff
@@ -423,14 +423,14 @@ def create_searchable_fields_and_values
 
       it "is usable as filter" do
         create_searchable_fields_and_values
-        id = WorkPackageCustomField.find_by_name("Searchable Field").id
+        id = WorkPackageCustomField.find_by(name: "Searchable Field").id
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.